jyulb/internal/isothermal/solver.pyx

```diff
@@ -0,0 +1,108 @@
+import cython
+import numpy as np
+cimport numpy as cnp
+from jyulb.internal.common.domain cimport PyAbstractIsothermalData
+
+
+STOKES_FLOW_ENUM = STOKES_FLOW
+LAMINAR_FLOW_ENUM = LAMINAR_FLOW
+TURBULENT_FLOW_ENUM = TURBULENT_FLOW
+
+BGK_ENUM = BGK
+TRT_ENUM = TRT
+MRT_ENUM = MRT
+REG_ENUM = REG
+
+
+cdef class PyFlowParams:
+
+    def __cinit__(self):
```

Comment:
The cdef for the class variable (a struct in c) is in solver.pxd: is this the correct way?